spec/lib/evm_database_spec.rb

```diff
@@ -29,4 +29,145 @@
       expect(described_class.seedable_model_class_names).to eq(%w(a z c))
     end
   end
+
+  context "schema checking" do
+    def stub_test_database(db_hash)
+      conn = double(:connection)
+      allow(conn).to receive(:tables).and_return(db_hash.keys)
+      db_hash.each do |table, col_names|
+        col_objs = col_names.map { |c| double(:name => c) }
+        allow(conn).to receive(:columns).with(table).and_return(col_objs)
+      end
+      conn
+    end
+
+    def stub_expected_schema(schema_hash)
+      allow(File).to receive(:read).with(subject::SCHEMA_FILE).and_return(YAML.dump(schema_hash))
+    end
+
+    describe ".write_expected_schema" do
+      it "writes the correct schema" do
+        test_db =
+          {
+            "table1" => %w(id data),
+            "table2" => %w(id uuid)
+          }
+        conn = stub_test_database(test_db)
+        expected_contents = "---\ntable1:\n- id\n- data\ntable2:\n- id\n- uuid\n"
+
+        expect(File).to receive(:write).with(subject::SCHEMA_FILE, expected_contents)
```

Comment:
This test isn't really testing anything new either way. All of the other tests will also go through `.current_schema` so I guess this could be removed if we would rather go that route.